### PR TITLE
bump ghodss/yaml and library-go

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -10,13 +10,13 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
...
 
 # fail early if any of the staging dirs is checked out
-for pkg in "$GOPATH/src/k8s.io/kubernetes/staging/src/k8s.io/"*; do
-  dir=$(basename $pkg)
-  if [ -d "$GOPATH/src/k8s.io/$dir" ]; then
-    echo "Conflicting $GOPATH/src/k8s.io/$dir found. Please remove from GOPATH." 1>&2
-    exit 1
-  fi
-done
+#for pkg in "$GOPATH/src/k8s.io/kubernetes/staging/src/k8s.io/"*; do
```
Comment (deads2k) [inline hack/update-deps.sh]: Revert this bit please

Comment (mfojtik) [reply hack/update-deps.sh]: i always doing this, almost feels like I want env var to override this

Comment (mfojtik) [reply hack/update-deps.sh]: fixed

**Comment by mfojtik** [comment]
/cc @smarterclayton 

This should sync us with kubernetes.

**Comment by mfojtik** [comment]
self-tagged, conflict

**Pull request merged**